Source/lua/randomat2/events/communist.lua

```diff
@@ -0,0 +1,98 @@
+AddCSLuaFile()
+
+local EVENT = {}
+
+EVENT.Title = "Communism! Time to learn how to share..."
+EVENT.id = "communist"
+
+
+util.AddNetworkString("AlertTriggerFinal")
+util.AddNetworkString("alerteventtrigger")
+
+local function RenameWeps(name)
+    if name == "sipistol_name" then
+        return "Silenced Pistol"
+    elseif name == "knife_name" then
+        return "Knife"
+    elseif name == "newton_name" then
+        return "Newton Launcher"
+    elseif name == "tele_name" then
+        return "Teleporter"
+    elseif name == "hstation_name" then
+        return "Health Station"
+    elseif name == "flare_name" then
+        return "Flare Gun"
+    elseif name == "decoy_name" then
+        return "Decoy"
+    elseif name == "radio_name" then
+        return "Radio"
+    elseif name == "polter_name" then
+        return "Poltergeist"
+    elseif name == "vis_name" then
+        return "Visualizer"
+    elseif name == "defuser_name" then
+        return "Defuser"
+    elseif name == "stungun_name" then
+        return "UMP Prototype"
+    elseif name == "binoc_name" then
+        return "Binoculars"
+    else
+        return name
+    end
+end
+
+function TriggerAlert(item, role, ply)
+    net.Start("alerteventtrigger")
+    net.WriteString(item)
+    net.WriteString(role)
+    net.Send(ply)
+end
+
+function EVENT:Begin()
+
+    hook.Add("TTTOrderedEquipment", "RandomatCommunismHook", function(ply, item, isItem)
+
+        local role = 0
+
+        if ply:GetRole() == ROLE_TRAITOR then
```

Comment:
Consider extracting this `if` to a common function either in the Randomat namespace or the Event namespace so it can be shared with the "privacy" event